### ISSUE TITLE
Scribble crashes when using solc imports

### ISSUE THREAD

**Issue by Jofairden** [ConsenSys/scribble#207]
When using imports in contracts I get an error when scribble is trying to resolve the annotations within the contract.
Imports in the contract are using forward slashes but the path separator is a backward slash.
Using an import of a contract is causing this error:

```
C:\Users\lingu\OneDrive\Documents\tests\node_modules\eth-scribble\dist\instrumenter\annotations.js:210
        text: raw.extractSourceFragment(source.contents),
                                               ^

TypeError: Cannot read properties of undefined (reading 'contents')
    at findAnnotations (C:\Users\lingu\OneDrive\Documents\tests\node_modules\eth-scribble\dist\instrumenter\annotations.js:210:48)
    at extractAnnotations (C:\Users\lingu\OneDrive\Documents\tests\node_modules\eth-scribble\dist\instrumenter\annotations.js:240:25)
    at buildAnnotationMap (C:\Users\lingu\OneDrive\Documents\tests\node_modules\eth-scribble\dist\instrumenter\annotations.js:465:31)
    at C:\Users\lingu\OneDrive\Documents\tests\node_modules\eth-scribble\dist\bin\scribble.js:610:61
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
   ```
   
   When adding comments to the scribble.js file, I found out that it is recognizing the import as a separate unit, and using that path within the contentsMap.
  
  When using a
  
  ```
  import {MyContract} from "./MyContract.sol";
  ```
  
  Results in:
  
  ```
  setting map value: C:\Users\lingu\OneDrive\Documents\tests\contracts/MyContract.sol with resolved path: C:\Users\lingu\OneDrive\Documents\tests\contracts/MyContract.sol
... <other map values here> ...
  setting map value: C:\Users\lingu\OneDrive\Documents\tests\contracts\MyContract.sol with resolved path: C:\Users\lingu\OneDrive\Documents\tests\contracts\MyContract.sol
  ```
Notice the change to the forward slash, which makes this path incompatible. Later down the chain this code is failing because the source is null.

# versions
Truffle v5.7.1 (core: 5.7.1)
Ganache v7.6.0
Solidity - 0.8.17 (solc-js)
Node v16.15.1
Web3.js v1.8.1
Scribble v0.6.16

**Comment by cd1m0** [comment]
Thanks for the report! If I understand correctly, the issue here is Scribble not correctly handling windows paths? This might take me a bit of time to reproduce/fix as I need to get a windows dev environment setup. Will see what I can do :)

**Comment by cd1m0** [comment]
Should be fixed with version 0.6.18 (fixed in PR https://github.com/ConsenSys/scribble/pull/210). If the issue is not resolved, please feel free to re-open this.
